ringtail/storagemanager.py

```diff
@@ -350,18 +266,18 @@ def filter_results(
         )  # make sure we keep Pose_ID in view
 
         self._insert_bookmark_info(self.current_view_name, view_query, all_filters)
-        
+
         # perform filtering
         if suppress_output:
             return None
-        
+
         logger.debug("Running filtering query...")
         time0 = time.perf_counter()
-        filtered_results = self._run_query(filter_results_str)
+        filtered_results = self._run_query(filter_results_str)  # TODO problem
```

Comment:
this TODO seems a little vague?

Reply:
Yes, this was left over from debugging and is removed